go/libraries/doltcore/sqle/schema_table_test.go

```diff
@@ -45,10 +47,47 @@ func TestAncientSchemaTableError(t *testing.T) {
 	}), sql.Collation_Default, "")
 	require.NoError(t, err)
 
-	_, _, err = db.GetTableInsensitive(ctx, doltdb.SchemasTableName)
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "cannot migrate dolt_schemas table from v0.19.1 or earlier")
+	sqlTbl, found, err := db.GetTableInsensitive(ctx, doltdb.SchemasTableName)
+	require.NoError(t, err)
```

Comment:
(minor) Probably worth updating the test name from `TestAncientSchemaTableError` to something like `TestAncientSchemaTableMigration` now that it doesn't test for an error any more.